kind-config.yaml

```diff
@@ -0,0 +1,7 @@
+# three node (two workers) cluster config
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+- role: worker
+  replicas: 3
```

Comment:
Out of sync from the comment; do we want 3 workers or 2?

Reply:
3 is a typical cluster that gives us enough workers I think for a test.

Reply:
👍 thats fine; I just didnt want the comment there saying 2 if we were setting 3.